### PR TITLE
added disclaimer for broken tools

### PR DESCRIPTION
**Pull request by cpholguera**
As discussed in https://github.com/OWASP/owasp-mstg/pull/1455#discussion_r326864653

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit dda62f5232e7e86406c4aefa32d3e4c979f400f9: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x08-Testing-Tools.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ To perform security testing different tools are available in order to be able to
 
 > The MSTG project has no preference in any of the tools below, or in promoting or selling any of the tools. All tools below have been verified if they are "alive", meaning that updates have been pushed recently. Nevertheless, not all tools have been used/tested by the authors, but they might still be useful when analyzing a mobile app. The listing is sorted in alphabetical order. The list is also pointing out commercial tools.
 
+> Disclaimer: At the time of writing, we ensure that the tools being used in the MSTG examples are properly working. However, the tools might be broken or not work properly depending on your OS version, if you're using a rooted/jailbroken device, the specific version of the rooting/jailbreak method and or the version of the tool. The MSTG does not take any responsibility over the working status of the tools. If you find a broken tool or example, please search or file an issue in the tool original page, e.g. in the GitHub issues page.
```
Comment (commjoen) [inline Document/0x08-Testing-Tools.md]:
```suggestion
> Disclaimer: At the time of writing, we ensure that the tools being used in the MSTG examples are properly working. However, the tools might be broken or not work properly depending on your OS version of both your host-computer and your test-device. The functioning of the tooling can be further impeded by whether you're using a rooted/jailbroken device, the specific version of the rooting/jailbreak method and or the version of the tool. The MSTG does not take any responsibility over the working status of the tools. If you find a broken tool or example, please search or file an issue in the tool original page, e.g. in the GitHub issues page.
```

Comment (cpholguera) [reply Document/0x08-Testing-Tools.md]: I've slightly edited the suggestion so that I can commit it from here.

**Comment by OMTGreleaser** [comment]
Results for commit f174f4e227a79ea95d5959931d8a8b8f18f05c8b: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**